docs/development/creating-api.md

```diff
@@ -138,7 +138,7 @@ interface Process {
 At the very bottom of your `api_name.cc` file:
 
 ```cpp title='api_name.cc'
-NODE_LINKED_MODULE_CONTEXT_AWARE(electron_browser_{api_name},Initialize)
+ELECTRON_BROWSER_MODULE({api_name}, Initialize)
```

Comment:
Might make sense to document all the 3 macros?

Reply:
I was thinking about it, but the example only mentions adding a browser binding.

Reply:
Fair point, can done in a separate PR then :+1: